11_circular_csimeone/README.md

```diff
@@ -0,0 +1,20 @@
+# Circular Distributions of CONUS Wide Streamflow
```

Comment:
Include a section here describing where the data came from and a short description of the data processing steps that were done

Reply:
I've added some info here